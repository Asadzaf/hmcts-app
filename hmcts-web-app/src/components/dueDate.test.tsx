import '@testing-library/jest-dom';

import { render } from '@testing-library/react';
import DueDate from './dueDate';

test('renders Header correctly', () => {
  const { asFragment } = render(<DueDate onDateChange = {jest.fn}/>);
  expect(asFragment()).toMatchSnapshot();
});
