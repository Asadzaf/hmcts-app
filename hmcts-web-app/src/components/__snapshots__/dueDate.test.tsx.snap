// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header correctly 1`] = `
<DocumentFragment>
  <div
    class="govuk-form-group "
  >
    <fieldset
      class="govuk-fieldset"
      role="group"
    >
      <legend
        class="govuk-fieldset__legend"
      >
        Due Date
      </legend>
      <div
        class="govuk-date-input"
        id="due-date"
      >
        <div
          class="govuk-date-input__item"
        >
          <label
            class="govuk-label govuk-date-input__label"
            for="due-date-day"
          >
            Day
          </label>
          <input
            class="govuk-input govuk-date-input__input govuk-input--width-2"
            id="due-date-day"
            inputmode="numeric"
            name="day"
            pattern="[0-9]*"
            type="text"
            value=""
          />
        </div>
        <div
          class="govuk-date-input__item"
        >
          <label
            class="govuk-label govuk-date-input__label"
            for="due-date-month"
          >
            Month
          </label>
          <input
            class="govuk-input govuk-date-input__input govuk-input--width-2"
            id="due-date-month"
            inputmode="numeric"
            name="month"
            pattern="[0-9]*"
            type="text"
            value=""
          />
        </div>
        <div
          class="govuk-date-input__item"
        >
          <label
            class="govuk-label govuk-date-input__label"
            for="due-date-year"
          >
            Year
          </label>
          <input
            class="govuk-input govuk-date-input__input govuk-input--width-4"
            id="due-date-year"
            inputmode="numeric"
            name="year"
            pattern="[0-9]*"
            type="text"
            value=""
          />
        </div>
      </div>
    </fieldset>
  </div>
</DocumentFragment>
`;
